plugins/configuration.rb

```diff
@@ -175,6 +175,7 @@ def render(context)
             platform: platform,
             converter: converter
           )}
+          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/"><i class="icon-question" />Find your configuration</a>
```

Comment:
```suggestion
          <p class="configuration-link" title="Click here to find your configuration file"><a href="/docs/configuration/" target="_blank"><i class="icon-question" />Find your configuration</a>
```

Currently, it takes over the current tab, maybe it's useful to have it open in a new tab to not disturb the user's "flow"?

Reply:
I think that is wise in this case (the new tab) 👍 

Unfortunately, this configuration block is "misused" in some places at this point still. But that should not stop this PR. Hopefully they will bubble up :)

Reply:
For those places a link might make sense anyway, if not I'm sure we will get feedback on it 👍